Firestore/core/src/firebase/firestore/remote/exponential_backoff.h

```diff
@@ -33,12 +33,12 @@ using Milliseconds = util::AsyncQueue::Milliseconds;
  * Initial backoff time in milliseconds after an error. Set to 1s according to
  * https://cloud.google.com/apis/design/errors.
  */
-const Milliseconds kDefaultBackoffInitialDelay = Milliseconds(1000);
+static const Milliseconds kDefaultBackoffInitialDelay = Milliseconds(1000);
 
-const double kDefaultBackoffFactor = 1.5;
+static const double kDefaultBackoffFactor = 1.5;
 
 /** Maximum backoff time in milliseconds. */
-const Milliseconds kDefaultBackoffMaxDelay = Milliseconds(60 * 1000);
+static const Milliseconds kDefaultBackoffMaxDelay = Milliseconds(60 * 1000);
```

Comment:
FWIW I'm not sure if `static` is necessary / useful here.  I suggested it in the other case because in a `.m` file, `static` is [necessary](https://stackoverflow.com/questions/1811041/const-vs-static-nsstrings-in-objective-c) to make sure the variable is scoped to just that `.m` file.  That said: a) `static` is [the default for .cc files](https://stackoverflow.com/questions/3709207/c-semantics-of-static-const-vs-const), b) I'm not sure about .mm files, c) since this is a .h file, I think [static is the default here as well](https://stackoverflow.com/a/4817981) and so adding it doesn't do anything?

All that said, it would probably be better to define these as public members of the ExponentialBackoff class if we're going to expose them.  E.g. similar to [Query::kNoLimit](https://github.com/firebase/firebase-ios-sdk/blob/1553801fec6ebd93804b56b701ad0b529bf49de3/Firestore/core/src/firebase/firestore/core/query.h#L51)

Clearly I don't know how to C++ very well.  So feel free to weight for Gil to weigh-in on if/how to expose these properly in C++. :-)